internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -103,7 +112,7 @@ func (s *CreateRuntimeResourceStep) Run(operation internal.Operation, log logrus
 			log.Infof("Runtime resource already created %s/%s: ", operation.KymaResourceNamespace, runtimeResourceName)
 			return operation, 0, nil
 		} else {
-			err := s.updateRuntimeResourceObject(runtimeCR, operation, runtimeResourceName, kymaResourceName, kymaResourceNamespace)
+			err := s.updateRuntimeResourceObject(values, runtimeCR, operation, runtimeResourceName, operation.CloudProvider)
```

Comment:
Idea for the future: we could have minize duplications by keeping dryRun if as small as possible with share common parts for real and dry invocation in the same instructions.

Reply:
DryRun would be removed.

Reply:
This dryRun is as small as possible.
But it will be gone in a month.